R/render-plot.R

```diff
@@ -89,13 +89,24 @@ renderPlot <- function(expr, width='auto', height='auto', res=72, ...,
   session <- NULL
   outputName <- NULL
 
+  # The purpose of this cleanup trigger is to allow `drawReactive` to be GC'd
+  # when this renderPlot is no longer needed. (#2423)
+  cleanup_trigger <- reactiveVal(0)
+  cleanup <- function() {
+    cleanup_trigger(1)
+  }
```

Comment:
We had discussed implementing `cleanup` without introducing an additional `reactiveVal`, so as not to pollute reactlog further?